JungleMinds/JM_Assuline
# Added ContactForm component

#### What does this PR do?

- [X] Implemented ContactForm component using `formik` and `yup`
- [X] Used `SendGrid` to send emails
- [X] Added error icon
- [X] Updated icon typings
- [X] Added temporary util function to send request to 

#### How should this be tested?

The component is in index page. Set your own email address in `.env.*` files and fill in the form and submit. You should get an email with form content.

#### Any background context you want to provide?

The form functionality doesn't work in IE for now.

#### What are the relevant tickets?

#### Definition of Done (remove if not applicable):

There are problems with IE to be solved
- [ ] No (offensive) mock data is used
There is mock data in index page
